### DISCUSSION TITLE
My NetBeans does not have the Hibernate configuration file wizard, how can I add this?

### DISCUSSION THREAD

**Discussion by evanr1234** [community/community#142791] [Programming Help]
### Body

I downloaded Apache NetBeans 23, and Java 23. However, when I click on the projects tab/right click the source packages/new/other there is no Hibernate category for me to click to use the configuration file wizard.

It never contained hibernate in the Java Dependencies folder, I added it manually by adding the dependency to the pom.xml file and I have the Hibernate and Jakarta JARS in my dependencies folder.

I am able to import and use the Hibernate and Jakarta packages into my project code.

How can I add the configuration file wizard functionality?

I made the configuration file manually but it has not worked yet.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by EliasDeHondt** [comment]


**Reply by lolo-cs44** [reply]
hi sorry for bothering you but I have a problem when I add  a class java to the same project it becomes gray and a package is this problem cross to you ?
![image](https://github.com/user-attachments/assets/015db8c4-3eb5-45aa-b4d6-22606b9c481d)

**Comment by Pnkajkanojiya** [comment]
thanks for suggestion
